Source/WebCore/Modules/model-element/HTMLModelElement.cpp

```diff
@@ -500,6 +501,8 @@ void HTMLModelElement::attributeChanged(const QualifiedName& name, const AtomStr
         updateLoop();
     else if (name == environmentmapAttr)
         updateEnvironmentMap();
+    else if (name == noportalAttr)
```

Comment:
You need to check whether NonPortaledModelEnabled is enabled or not here.

Reply:
Thanks. Updated to use runtime check.